src/kernelci_pipeline/logspec_api.py

```diff
@@ -114,7 +114,10 @@ def new_issue(logspec_error, object_type):
         'comment': comment,
         'misc': {
             'logspec': error_copy
-        }
+        },
+        # Set culprit_code to True by default
+        # OBS: this needs to be reviewed on every logspec upgrade
+        'culprit': {'code': True}
```

Comment:
You also gotta set the rest of the members to false. Otherwise their status is considered unknown. That is here you're saying: "I know this is a kernel issue, but it also might be a test issue or a CI issue at the same time".

Reply:
Fixed, please check https://grafana.kernelci.org/d/issue/issue?var-version=0&var-id=maestro:bba672c389cd8e575694e468d87a80b1b4074241&var-datasource=edqupth5xi8e8b